pkg/proxy/ipvs/testing/fake_test.go

```diff
@@ -17,33 +17,36 @@ limitations under the License.
 package testing
 
 import (
-	"reflect"
 	"testing"
 
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/kubernetes/pkg/proxy/ipvs"
 )
 
 func TestSetGetLocalAddresses(t *testing.T) {
 	fake := NewFakeNetlinkHandle()
+	_ = ipvs.NetLinkHandle(fake) // Ensure that the interface is honored
```

Comment:
Good point. Moved.

Reply:
Moved back since:
```
ipvs > go test
# k8s.io/kubernetes/pkg/proxy/ipvs
package k8s.io/kubernetes/pkg/proxy/ipvs
        imports k8s.io/kubernetes/pkg/proxy/ipvs/testing: import cycle not allowed in test
```